### PR TITLE
Speed up loading books on dev homepage

### PR DESCRIPTION
**Pull request by PiotrKozlowski**
There is no card for this. It is just something that was annoying for me. I don't want to wait for all books to load because I'm usually testing on American Government 2e

for: openstax/unified#1572

### CODE REVIEW

**Comment by jarosik10** [comment]
Yay, I'd be helpful. You have to add/change some tests for 100% coverage. ;)

File: src/app/developer/components/Books.tsx [hunk 1]
```diff
@@ -24,11 +24,22 @@ class Books extends React.Component<Props, State> {
     const {archiveLoader, osWebLoader} = this.props.services;
     const bookLoader = makeUnifiedBookLoader(archiveLoader, osWebLoader);
 
-    const books = await Promise.all(Object.entries(BOOKS).map(([bookId, {defaultVersion}]) =>
-      bookLoader(bookId, defaultVersion)
-    ));
-
-    this.setState({books});
+    const sortBooks = (bookA: Book, bookB: Book) => {
```
Comment (TomWoodward) [inline src/app/developer/components/Books.tsx:27]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare does this in one line

**Review by TomWoodward — CHANGES_REQUESTED** [review]
if you use the other url https://rex-web.herokuapp.com/books/list the sw caches the requests and its way faster

**Comment by PiotrKozlowski** [comment]
@jarosik10 
> Yay, I'd be helpful. You have to add/change some tests for 100% coverage. ;)

Done

**Review by jarosik10 — APPROVED** [review]

**Review by TomWoodward — APPROVED** [review]

**Pull request merged**